<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回流焊仿真软件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1,1,1,1">
     <item>
      <widget class="QLabel" name="title">
       <property name="text">
        <string>回流焊仿真软件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="argumentPage">
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modelPage">
       <property name="text">
        <string>模型设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataProcessPage">
       <property name="text">
        <string>数据处理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitSystem">
       <property name="text">
        <string>退出系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>模型训练</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,20">
        <item>
         <widget class="QPushButton" name="datasetSelectBtn">
          <property name="text">
           <string>选择数据集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="datasetPath">
          <property name="text">
           <string>暂未导入文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>元件名称</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="compomentBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>贴片电阻</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>贴片电容</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>塑封芯片</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>陶封芯片</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="title">
           <string>温度区域</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QComboBox" name="placeBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>中心温度</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>引脚温度</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="modelBtn">
        <property name="text">
         <string>开始拟合</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
